mineral-ui/mineral-ui
# [Button]: Update default styles

### Description
* Update tokens per new default Button styles and disabled Button styles
    * Add backgroundColor
    * Update backgroundColor_disabled 
    * Update backgroundColor_focus
* Update Button with new default styles and refactor with new RTL property pattern
* Update ButtonGroup: new focus styles for border colors
    * Also add comments for complex selectors
* Update MenuItem: affected due to tokens change

### Motivation and context
Closes #697 

### Screenshots, videos, or demo, if appropriate

https://697-update-button--mineral-ui.netlify.com/components/button

### How to test


1. `cd mineral-ui && npm start`
1. Navigate to Button > [Default Button section](https://697-update-button--mineral-ui.netlify.com/components/button/default)
    1. Make sure all styles are appropriate
1. Navigate to [ButtonGroup](https://697-update-button--mineral-ui.netlify.com/components/button-group)
    1. Make sure all states style appropriately for the various combinations button types and props
1. Review the Happo diffs and many updated snapshots

### Types of changes
- New feature: New default and disabled styles for Button

### Checklist

### How does this PR make you feel?
![Man laying with 90s flowing clothes](https://media.giphy.com/media/l1K9EYgSVXfzviMkE/giphy.gif?cid=3640f6095babe56f6765736b73289aed)
